### DISCUSSION TITLE
유니티 AR 사물 인식을 위한 리소스 문의

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/XRStudy#31] [7. Q&A] [answered]
안녕하세요. 유니티 첫 프로젝트로 AR 사물 인식 시작으로 추적, 정보 띄우기, 네트워크 통신을 구현해보려고 합니다. 
관련하여서 참고하면 좋은 리소스가 있다면 답변 부탁드립니다. 

우선은 유니티 사이트에서
1. https://learn.unity.com/project/get-started-with-ar
2. https://learn.unity.com/project/create-a-marker-based-ar-app?language=en
가 AR 구현에 적합해보여서 따라 해볼 생각입니다. 

감사합니다.

**Comment by jongfeel** [comment, marked as answer]
좋습니다.

따라해 보면서 만들어야 하는 기능에 맞춰서 프로젝트에 적용해 보는 걸 추천드립니다.
그 내용은 이제 부터 issue를 적어서 관리해 보면 좋을 겁니다.
당장 적용해서 만들지 못하더라도, to do를 한다는 생각으로 적어 보는 것도 좋습니다.

Issue는 해야 할 일과 적용해야 하는 내용등을 적고
그 내용을 바탕으로 실제 작성한 코드를 연결시키는 방식으로 하기에 좋은 도구입니다.
너무 큰 단위로 적으면 한 번에 처리하기 힘드니까 
여러 단위로 나눠서 관리하는 방법도 이후에 알려드리겠습니다.

---

차례대로 진행하고 관리하고 적용하는 방법을 알려드릴테니
너무 조급해하거나 모른다고 그냥 있지 마시고
계속 질문/답변을 통해 알아가면 좋겠습니다.
